utils-testing-coroutines/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1 @@
+<manifest package="pm.gnosis.tests.utils.coroutines" />
```

Comment:
Maybe we don't need a manifest and we could make this module `kotlin/Java`  only from the get-go?

Reply:
Future PR?

Reply:
Sure :) we can make this part of a bigger effort to decouple the android bits from the kotlin bits :)